### PR TITLE
typo: Double word "the"

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @nschonni 
#sign-off

File: sccm/develop/core/understand/windows-powershell-basics.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ System Center 2012 Configuration Manager SP1 introduced native PowerShell suppor
 
  To get started:  
 
-1.  Hit your Windows key and type “PowerShell��?.  
+1.  Hit your Windows key and type "PowerShell".
```
Comment (nschonni) [inline sccm/develop/core/understand/windows-powershell-basics.md]: Thanks! think there are a bunch of smartquote/encoding issues in the repo if you do a search

**Comment by PRMerger15** [comment]
@ktoliver : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Pull request merged**